packages/helper/src/email/types/IEmail.ts

```diff
@@ -3,5 +3,9 @@ export interface IEmail {
 	to: string;
 	subject: string;
 	message: string;
-	attachments?: unknown;
```

Comment:
@Jacobjeevan please use 
```ts 
attachments?: any;
```
instead of unknown, unknown has side effects which cause errors
i've gone ahead and typed it for you

Reply:
@Enubia 👍